services/paymaster/src/config_manager/load_config.rs

```diff
@@ -0,0 +1,61 @@
+use arc_swap::ArcSwap;
+use std::collections::HashMap;
+use std::sync::Arc;
+use std::time::Duration;
+
+use crate::api::{self, DomainState};
+use crate::config_manager::config::Config;
+
+use crate::db;
+use anyhow::Result;
+use config::File;
+use tokio::time::interval;
+
+/// Load the config from the file.
+pub fn load_file_config(config_path: &str) -> Result<Config> {
+    let config: Config = config::Config::builder()
+        .add_source(File::with_name(config_path))
+        .build()?
+        .try_deserialize()?;
+    // note we DO NOT want to assign defaults here because this will be used to seed the database and there's no point adding the txs from eg. enable_session_management
```

Comment:
i don't like seeding the db here from file. i am not saying this is strictly wrong; i am just saying i don't think that we should be doing this here in the server. we can host this elsewhere rather than creating code for a one-time operation from the server that we probably won't ever want to run again.